Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h

```diff
@@ -616,7 +616,11 @@ compute_vertex_normal_as_sum_of_weighted_normals(typename boost::graph_traits<Po
 /**
 * \ingroup PMP_normal_grp
 *
-* computes the unit normal at vertex `v` as the average of the normals of incident faces.
+* \brief computes the unit normal at vertex `v` as a function of the normals of incident faces.
+*
+* The implementation is inspired by Aubry et al. "On the most 'normal' normal" \cgalCite{cgal:al-otmnn-08}.
+* Alternatively, if the most visible normal does not exist or is not found, the normal is computed
+* as a sin-weighted sum of the normals of the incident faces.
```

Comment:
Wouldn't it be clearer and better to use `sinus` instead of `sin` here?

Reply:
Out of sin/sine/sinus, "sine weighted" has the most hits on google scholar; "sin weighted" has most hits on google; "sinus weighted" is much lower for both.

Reply:
Indeed `sine` would be better.